<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personwindow</class>
 <widget class="QWidget" name="personwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Day of dirth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSex">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Male</string>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEditSurname">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDayB">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Day:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Month:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMonthB">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>January</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>February</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>March</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>April</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>June</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>July</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>August</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>September</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>October</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>November</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>December</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEditPlace">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>580</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxYearB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2008</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2007</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2006</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2003</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1999</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1998</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1997</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1996</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1995</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1994</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1993</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1992</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1991</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1990</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1989</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1988</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1987</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1986</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1985</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1984</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1983</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1982</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1981</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1980</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1979</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1978</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1977</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1976</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1975</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1974</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1973</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1972</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1971</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1970</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1969</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1968</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1967</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1966</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1965</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1964</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1963</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1962</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1961</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1960</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1959</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1958</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1957</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1956</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1955</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1954</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1953</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1952</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1951</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1950</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1949</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1948</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1947</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1946</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1945</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1944</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1943</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1942</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1941</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1940</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1939</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1938</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1937</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1936</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1935</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1934</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Change image</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditName_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
